flixel/FlxCamera.hx

```diff
@@ -51,11 +51,24 @@ class FlxCamera extends FlxBasic
 	public static var defaultZoom:Float;
 
 	/**
-	 * Which cameras a `FlxBasic` uses to be drawn on when nothing else has been specified.
-	 * By default, this is just a reference to `FlxG.cameras.list` / all cameras, but it can be very useful to change.
-	 * This may be temporarily modified during a `draw` call.
+	 * Used behind-the-scenes during the draw phase so that members use the same default
+	 * cameras as their parent.
+	 * 
+	 * Prior to 4.8.2 it was useful to change this value, but that feature is deprecated.
+	 * Instead use `FlxG.cameras.setDrawsDefault`.
 	 */
-	public static var defaultCameras:Array<FlxCamera>;
+	@:deprecated("`FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDrawsDefault()` instead")
+	public static var defaultCameras(get, set):Array<FlxCamera>;
+	
+	/**
+	 * Used behind-the-scenes during the draw phase so that members use the same default
+	 * cameras as their parent.
+	 * 
+	 * This is the non-deprecated list that the public `defaultCameras` proxies. Allows flixel classes
+	 * to use it without warning.
+	 */
+	@:allow(flixel)
```

Comment:
`@:allow(flixel)` is dangerous, at least it was in older Haxe versions, I don't remember whether it was changed - it would make it so that everything that inherits from a class in the `flixel` package has access too, which is typically pretty much everything... You can check whether that's the case via completion.

Bottom line, usually it's safer to use `@:access`.

Reply:
I personally don't like @:access, I prefer a fields to declare their intended access in the declaration. I didn't know it allowed access to extensions. I can change this to allow the specific methods.